src/docs/platform-architecture-reference/nsx-networking.md

```diff
@@ -18,24 +18,27 @@ content_owner: Steven Barre
 sort_order: 
 ---
 
-# NSX SDN Cluster <!-- omit in toc -->
```

Comment:
`<!-- omit in toc -->` is used by my VSC markdown plugin to auto update the table of contents list. https://marketplace.visualstudio.com/items?itemName=yzhang.markdown-all-in-one 

Can we keep that in there?

Reply:
Hmmm, is there any way to use that same comment to get your VS Code but without it being inline with the heading?

I've pruned anything like that from this repo because I'm using the [gatsby-remark-autolink-headers plugin](https://www.gatsbyjs.com/plugins/gatsby-remark-autolink-headers/) to auto-generate IDs with links on headings, and the IDs get mangled when we have any cruft inline with the headings.

Here's an example of what that heading ID turns into when I add the `<!-- omit in toc -->` comment back:

<img width="1840" alt="NSX networking page heading generation example" src="https://user-images.githubusercontent.com/25143706/190700161-9d1fa551-366d-4c4f-b5a7-69b193eb6522.png">

It's not the end of the world for the H1 heading, since that one is unlikely to ever be used, but for any other headings this kind of thing will break the auto-generated ToC in Gatsby (see [Compare Gatsby and GitHub heading ID generation](https://github.com/bcgov/platform-developer-docs/issues/113)).

For what it's worth, I have thought about removing all the H1 headings from the Markdown source docs and just generating this from the `title` frontmatter field, but then anyone looking at the doc on GitHub wouldn't have the title there. In general, my approach has been to focus on the Gatsby rendered site experience vs GitHub, but I thought it might be better for search inside of GitHub to leave those H1 headings in the Markdown.

Reply:
Missed that in those docs, thanks for pointing that out!